<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>strikeWaterDialog</class>
 <widget class="QDialog" name="strikeWaterDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>694</width>
    <height>303</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Strike Water Calculator</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>25</number>
   </property>
   <property name="topMargin">
    <number>9</number>
   </property>
   <property name="rightMargin">
    <number>25</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="header">
     <property name="topMargin">
      <number>5</number>
     </property>
     <property name="bottomMargin">
      <number>5</number>
     </property>
     <item>
      <widget class="QLabel" name="initialInfusionLbl">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Initial Infusion</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="mashInfusionLbl">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Mash Infusion</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="inputs">
     <item>
      <layout class="QVBoxLayout" name="initialInfusion">
       <item>
        <layout class="QHBoxLayout" name="grainTemp">
         <item>
          <widget class="QLabel" name="grainTempLbl">
           <property name="text">
            <string>Original Grain Temperature</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="grainTempVal">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="targetMash">
         <item>
          <widget class="QLabel" name="targetMashLbl">
           <property name="text">
            <string>Target Mash Temperature</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="targetMashVal">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="grainWeightInit">
         <item>
          <widget class="QLabel" name="grainWeightInitLbl">
           <property name="text">
            <string>Weigth of Grain</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="grainWeightInitVal">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="decimals" stdset="0">
            <number>1</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="waterVolume">
         <item>
          <widget class="QLabel" name="waterVolumeLbl">
           <property name="text">
            <string>Volume of Water</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="waterVolumeVal">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="infusionsVline">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="mashInfusion">
       <item>
        <layout class="QHBoxLayout" name="mashVol">
         <item>
          <widget class="QLabel" name="mashVolLbl">
           <property name="text">
            <string>Total Volume of Water</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="mashVolVal">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="grainWeight">
         <item>
          <widget class="QLabel" name="grainWeightLbl">
           <property name="text">
            <string>Grain Weight</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="grainWeightVal">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="actualMash">
         <item>
          <widget class="QLabel" name="actualMashLbl">
           <property name="text">
            <string>Actual Mash Temperature</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="actualMashVal">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="targetMashInf">
         <item>
          <widget class="QLabel" name="targetMashInfLbl">
           <property name="text">
            <string>Target Mash Temperature</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="targetMashInfVal">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="infusionWater">
         <item>
          <widget class="QLabel" name="infusionWaterLbl">
           <property name="text">
            <string>Infusion Water Temperature</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="infusionWaterVal">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="results">
     <item>
      <layout class="QHBoxLayout" name="initialResult">
       <item>
        <widget class="QLabel" name="initialResultLbl">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Strike Water Temperature</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="initialResultTxt">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="mashResult">
       <item>
        <widget class="QLabel" name="mashResultLbl">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Volume to add</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="mashResultTxt">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="unitsHline">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="leftMargin">
      <number>50</number>
     </property>
     <property name="rightMargin">
      <number>50</number>
     </property>
     <item>
      <widget class="QLabel" name="Notes">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Note: This calculator assumes a pre-heaten mash tun.</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_calculate">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Maximum">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Calculate</string>
       </property>
       <property name="text">
        <string>Calculate</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="noteHline">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
